ogr/services/github/project.py

```diff
@@ -277,20 +277,20 @@ def _get_collaborators_with_permission(self) -> dict:
             collaborators[user.login] = permission
         return collaborators
 
+    @indirect(GithubIssue.get_list)
     def get_issue_list(
         self,
         status: IssueStatus = IssueStatus.open,
         author: Optional[str] = None,
         assignee: Optional[str] = None,
         labels: Optional[List[str]] = None,
     ) -> List[Issue]:
-        return GithubIssue.get_list(
-            project=self, status=status, author=author, assignee=assignee, labels=labels
-        )
+        pass
```

Comment:
Is mypy `ok` with that? Will it still check if the attributes do not match?

Reply:
Seems to be catching it, just tested it